sklearn/tests/test_dummy.py

```diff
@@ -479,6 +480,22 @@ def test_constant_strategy_multioutput():
     _check_predict_proba(clf, X, y)
 
 
+def test_constant_strategy_y2d():
+    X = [[0], [0], [0], [0]]  # ignored
+    y = np.array([[2],
+                  [1],
+                  [2],
+                  [2]])
+
+    n_samples = len(X)
+
+    clf = DummyClassifier(strategy="constant", random_state=0,
+                          constant=[1])
+    clf.fit(X, y)
+    assert_array_equal(clf.predict(X), np.ones((n_samples, 1)))
```

Comment:
Hm I thought for multi-output it was different, but RandomForestClassifier is multi-output. But apparently there's no common test for that? Gah!